src/shared/constants/avatars.ts

```diff
@@ -0,0 +1,113 @@
+const avatars = [
```

Comment:
```suggestion
const AVATARS = [
```
Не изменяемое же значение

Reply:
Done